package.json

```diff
@@ -51,7 +51,7 @@
     }
   },
   "engines": {
-    "node": ">=14.17"
+    "node": ">=18"
```

Comment:
Is this bump required? If so, this needs to be merged as a `BREAKING CHANGE`.

Reply:
@tadhglewis this requires consumers of koala to use Node.js 18+.

skuba is a dev dependency. It doesn't impose a transitive requirement for packages to also require Node.js 18+ themselves.

Reply:
We could require Node.js 18+ if that's desired for other reasons, but that also feels separate from this Renovate PR.